cvlab/view/widgets.py

```diff
@@ -166,7 +166,7 @@ def wheelEvent(self, event):
 
     def resize_previews(self, new_size):
         out_size = np.clip(int(new_size), 64, 2048)
-        self.preview_size = out_size
+        self.preview_size = int(out_size)
```

Comment:
to dość duża zmiana, czy na pewno ją przemyślałeś? xd

Reply:
tak